### DISCUSSION TITLE
Default intervals for temperature and humidity reporting in Z03MMC_v0120.bin

### DISCUSSION THREAD

**Discussion by pnickv** [pvvx/ZigbeeTLc#75] [Q&A] [answered]
Hey there!
I flashed Z03MMC_v0120.bin on my both devices and they both send humidity reports every 60 seconds but temperature - every 60 _minutes_. Is it correct behavior and how it could be configured to the same interval? (I personally would prefer 60 minutes in order to minimize power consumption)

**Comment by pvvx** [comment]
Default intervals for temperature and humidity reporting are set to 30-180 (min-max) seconds, reportableChanged to 0.1C and 0.5%.

**Reply by pnickv** [reply]
Than you for the answer! I saw that information in [readme](https://github.com/pvvx/ZigbeeTLc?tab=readme-ov-file#current-additions). As i can understand these values are configured [here](https://github.com/pvvx/ZigbeeTLc/blob/1b96ae242590e87a0015add5d4587844264d01d2/src/zb_config.h#L92)? 
So I just downloaded Z03MMC_v0120.bin, flashed it without any changes and a bit confused with such a big difference between intervals for humidity and temperature. So I am not sure what is the reason for this difference

**Reply by pvvx** [reply, marked as answer]
No.
These values are set by the [Zigbee coordinator during binding](https://github.com/pvvx/ZigbeeTLc/issues/6).
Default values are used if the coordinator did not set their own during binding.
Learn Zigbee if you want to use Zigbee devices.

**Reply by pnickv** [reply]
Got it! Thank you for the explanation!
